cartography/rules/data/frameworks/mitre_attack/requirements/t1098_account_manipulation/__init__.py

```diff
@@ -71,16 +72,14 @@
     ),
     cypher_query="""
         MATCH (a:AWSAccount)-[:RESOURCE]->(principal:AWSPrincipal)
-        MATCH (principal)-[:POLICY]->(:AWSPolicy)-[:STATEMENT]->(stmt:AWSPolicyStatement)
-        OPTIONAL MATCH (groupmember:AWSUser)
+        MATCH (principal)-[:POLICY]->(policy:AWSPolicy)-[:STATEMENT]->(stmt:AWSPolicyStatement{effect:"Allow"})
```

Comment:
Should we have something in here that also accounts for Denies that override other Allows?